drivers/sensor/isl29125/isl29125.c

```diff
@@ -55,22 +55,22 @@ static void isl29125_write16(struct device *i2c_dev, u8_t reg, u16_t data)
 	}
 }
 
-static u16_t isl29125_readRed(struct device *i2c_dev)
+static u16_t isl29125_read_red(struct device *i2c_dev)
```

Comment:
Please amend the commit that added the driver instead of making a new commit. It's ok to rewrite history in a PR